client/tracing/src/lib.rs

```diff
@@ -57,7 +57,7 @@ const ZERO_DURATION: Duration = Duration::from_nanos(0);
 
 // The layered Subscriber as built up in `init_logger()`.
 // Used in the reload `Handle`.
-type SCSubscriber<
+pub type SCSubscriber<
```

Comment:
Can we limit the visibility of this type? It's not something we want anyone to use really.